README.md

```diff
@@ -1,3 +1,15 @@
+# :warning: App Search and Workplace Search will be discontinued in 9.0
+
+Starting with Elastic version 9.0, the standalone Enterprise Search products, will no longer be included in our offering.
+They remain supported in their current form in version 8.x and will only receive security upgrades and fixes.
+Enterprise Search clients will continue to be supported in their current form throughout 8.x versions, according to our [EOL policy](https://www.elastic.co/support/eol).
+We recommend transitioning to our actively developed [Elastic Stack](https://www.elastic.co/elastic-stack) tools for your search use cases. However, if you're still using any Enterprise Search products, we recommend using the latest stable release of the clients.
+
+Here are some useful links with more information:
+
+- [Enterprise Search FAQ](https://www.elastic.co/resources/enterprise-search/enterprise-search-faq)
+- [One stop shop for Upgrading to Elastic Search 9](https://www.elastic.co/guide/en/enterprise-search/current/upgrading-to-9-x.html)
```

Comment:
@leemthompo With the recent migration I believe this link changed. Could you please confirm the final link for the ent search upgrade docs?